.github/actions/invalidate_master_cache/action.yml

```diff
@@ -4,7 +4,7 @@ runs:
   using: composite
   steps:
     - name: save_cache
-      uses: actions/cache/save@v3
+      uses: actions/cache/save@13aacd865c20de90d75de3b17ebe84f7a17d57d2
```

Comment:
how are we generating these hashed versions?

Reply:
@shayan-deriv, This is the commit hash from the author's repo. Security team wants us to use a fixed version of the action, for more control I guess